src/components/ui/ListEntry.js

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+
+const LeftIcon = styled.div`
+  border-radius: 50%;
+  max-width: 36px;
+  max-height: 36px;
+  overflow: hidden;
+
+  & > * {
+    width: 100%;
+    height: 100%;
+  }
+`
+const PrimaryText = styled.div`
+  align-items: center;
+  font-weight: bold;
+  font-size: 14px;
+`
+const SecondaryText = styled.div`
+  align-items: center;
+  font-weight: normal;
+  font-size: 12px;
+`
+
+const ListContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 0 22px;
+  height: 57px;
+  align-items: center;
+`
+const TextContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  color: ${({ theme }) => theme.secondaryText};
+  justify-content: center;
+  margin-left: 18px;
+  flex: 1;
+`
+const RightIcon = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 32px;
+  height: 32px;
+  color: ${({ theme }) => theme.blue};
+`
+
+const ListEntry = ({ leftIcon, primaryText, secondaryText, rightIcon }) => (
```

Comment:
Thinking about this more: 

![image](https://user-images.githubusercontent.com/4369024/109697134-79267e80-7b53-11eb-84be-86b3e6a1c2f0.png)

This entry in the desktop design has multiple icons on the right. We can support that by taking `rightIcons` as a list of elements, then mapping them in the render. Same things with `leftIcons`.

Reply:
Additionally, we want to take a `size` prop in ListEntry that can be `sm` or `md`. Make a constant object (basically an enum) that maps `sm` and `md` to sizes for the height of ListEntry and also the left and right icons.

Reply:
Taking a set of discrete sizes, rather than numerical sizes, will come up a couple other times in our UI components. Most immediately, `IconButton` @laurenho025 @Archna-1 (sm, md, lg, xl).

Reply:
@Archna-1 @code-arman update here -- we decided against this. Pass a prop `size` as a number for the width/height (e.g. `<MyComponent size={40} />` for 40 x 40px)